src/components/markdown/index.jsx

```diff
@@ -0,0 +1,108 @@
+import React from "react";
+
+import PropTypes from "prop-types";
+import rehypeReact from "rehype-react";
+import styled from "styled-components";
+
+const PrimaryTitle = styled.h1`
+    font-weight: 400;
+    font-size: 2em;
+
+    @media (min-width: ${(props) => props.theme.breakpoints[1]}px) {
+        font-size: 3em;
+    }
+`;
+
+const SecondaryTitle = styled.h2`
+    font-family: "Source Sans Pro", sans-serif;
+    font-size: 1.75em;
+    font-weight: 600;
+    margin: calc(1.25em + 0.15vw) 0 calc(0.75em + 0.15vw);
+
+    @media (min-width: ${(props) => props.theme.breakpoints[1]}px) {
+        font-size: 2em;
+    }
+`;
+
+const TertiaryTitle = styled.h3`
+    font-family: "Source Sans Pro", sans-serif;
+    font-size: 1.5em;
+    font-weight: 600;
+    margin: calc(1em + 0.15vw) 0 calc(0.75em + 0.15vw);
+
+    @media (min-width: ${(props) => props.theme.breakpoints[1]}px) {
+        font-size: 1.5em;
+    }
+`;
+
+const Paragraph = styled.p`
+    font-size: calc(1em + 0.15vw);
+    line-height: 1.7em;
+    margin: 0 0 1em;
+    color: var(--text-gray);
+`;
+
+const Bold = styled.strong`
+    color: var(--text-dark-black);
```

Comment:
We are fragmenting colors in both css and js with theme.

My pov is that theme is a superset of css vars and should be used instead.

Reply:
I agree. I believe it's better to discuss this change in a separate PR.